File: src/WebSPA/ClientApp/src/app/shared/models/person.model.ts
import { Post } from '.';

export class Person {
    public personId: any;
    public userId: any;
    public username: string;
    public name: string;
    public description: string;
    public email: string;
    public isFollowing: boolean;
    public following: Person[];
    public followingCount: number;
    public followers: Person[];
    public followersCount: boolean;
    public posts: Post[];
    public postsCount: boolean;
    public profilePicture: string;
}

File: src/Services/FeedService/FeedService.API/Domain/Model/PersonModel.cs
using System;
using System.Collections.Generic;
using FeedService.Domain.States;

namespace FeedService.Domain.Model
{
    public class PersonModel
    {
        public Guid PersonId { get; set; }
        public Guid UserId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public IList<PersonModel> Following { get; set; } = new List<PersonModel>();
        public int FollowersCount => Followers.Count;
        public IList<PersonModel> Followers { get; set; } = new List<PersonModel>();
        public int FollowingCount => Following.Count;
        public IList<Post> Posts { get; set; } = new List<Post>();
        public int PostsCount => Posts.Count;
        
        
    }
}